### DISCUSSION TITLE
Win10  cargo run error ???  help !!!

### DISCUSSION THREAD

**Discussion by xialixx** [rustdesk/rustdesk#8351] [Q&A] [answered]
cargo run ！！！！！！！！！！

error: failed to run custom build command for `magnum-opus v0.4.0 (https://github.com/rustdesk-org/magnum-opus#5cd2bf98)`

Caused by:
  process didn't exit successfully: `D:\rust\rustdesk\target\debug\build\magnum-opus-d2d97819d6a580eb\build-script-build` (exit code: 101)
  --- stdout
  cargo:info=x64-windows-static
  cargo:rustc-link-lib=static=opus
  cargo:rustc-link-search=D:\rust\vcpkg\installed\x64-windows-static\lib
  cargo:include=D:\rust\vcpkg\installed\x64-windows-static\include
  rerun-if-changed=C:\Users\XL\.cargo\git\checkouts\magnum-opus-6f07f61b97016ae9\5cd2bf9\opus_ffi.h
  rerun-if-changed=D:\rust\vcpkg\installed\x64-windows-static\include

  --- stderr
  thread 'main' panicked at C:\Users\XL\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bindgen-0.59.2\src/lib.rs:2144:31:
  Unable to find libclang: "couldn't find any valid shared libraries matching: ['clang.dll', 'libclang.dll'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])"
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...

**Comment by rustdesk** [comment, marked as answer]
https://discord.com/invite/nDceKgxnkV build channel
